src/sagemaker/serve/builder/model_builder.py

```diff
@@ -493,6 +496,12 @@ def _model_builder_register_wrapper(self, *args, **kwargs):
         self.pysdk_model.model_package_arn = new_model_package.model_package_arn
         new_model_package.deploy = self._model_builder_deploy_model_package_wrapper
         self.model_package = new_model_package
+        if getattr(self, "_is_mlflow_model", False) and self.mode == Mode.SAGEMAKER_ENDPOINT:
```

Comment:
Where is `_is_mlflow_model` method define?

Reply:
https://github.com/aws/sagemaker-python-sdk/blob/2a902cd116fe0ecdeb9f0849f94a7eb49a7cf755/src/sagemaker/serve/builder/model_builder.py#L724